lib/Services/userService.dart

```diff
@@ -29,4 +29,21 @@ class UserService {
         ApiError.fromJson(jsonDecode(utf8.decode(response.bodyBytes)));
     throw HttpException(error.message.toString());
   }
+
+  Future<UserApp> fazrequest(String email, String password) async {
```

Comment:
Pq o nome dessa função? Ela está muito geral, se for para entrar com a lógica do login, coloca ela na mesma função que a chamada.

Reply:
Errei o nome e acabou ficando, agora ela virou signInWithEmailAndPassword()